packages/guilded.js/lib/structures/Server.ts

```diff
@@ -0,0 +1,81 @@
+import type { Client } from "./Client";
+import { Base } from "./Base";
+import type { ServerPayload, ServerType } from "@guildedjs/guilded-api-typings";
+import type { User } from "./User";
+import type { Channel } from "./channels";
+
+export class Server extends Base<ServerPayload> {
+    /** The ID of the owner of this server */
+    ownerId: string;
+    /** The type of this server */
+    type: ServerType | null;
+    /** The name of this server */
+    name!: string;
+    /** The URL this server is accessible from */
+    url!: string;
+    /** The description of this server */
+    description!: string | null;
+    /** The icon of this server */
+    iconURL!: string | null;
+    /** The banner of this server */
+    bannerURL!: string | null;
+    /** The timezone this server is in */
+    timezone!: string | null;
+    /** Whether this server is verified or not */
+    isVerified!: boolean;
+    /** The default channel of this server */
+    defaultChannelId!: string;
+    /** The date this server was created */
+    createdAt!: Date;
+
+    constructor(client: Client, data: ServerPayload) {
+        super(client, data);
+        this.ownerId = data.ownerId;
+        this.type = data.type ?? null;
+        this.createdAt = new Date(data.createdAt);
+        this._update(data);
+    }
+
+    get owner(): User | null {
+        return this.client.users.cache.get(this.ownerId) ?? null;
+    }
```

Comment:
should get the member object since this is a server
```suggestion
    get owner(): Member | null {
        return this.client.members.cache.get(buildMemberKey(this.id, this.ownerId)) ?? null;
    }
```
(need to import Member and buildMemberKey)